aiida/environment.yml

```diff
@@ -5,7 +5,6 @@ channels:
 dependencies:
 # - aiida-core
 - aiida-core.services
-- activate-aiida
 # while aiida-core.services requires rabbitmq-server>=3.7, for some reason version 3.6.15 is installed without the line below
-- rabbitmq-server>=3.8
+# - rabbitmq-server>=3.8
```

Comment:
not needed anymore?

Reply:
I think this stems from some peculiarities of past versions, which I am not familiar with. Now, just installing
`aiida-core.services` and `aiida-core` into the conda environment should be sufficient. I'll try it out with an updated
docker image, and with the `aiida-core` dependency moved here.